avm/res/insights/action-group/README.md

```diff
@@ -242,6 +242,21 @@ module actionGroup 'br/public:avm/res/insights/action-group:<version>' = {
         }
       ]
     },
+    "incidentReceivers": {
+      "value": [
+        {
+          "connection": {
+            "id": "0",
+            "name": "MyConnectionName"
+          },
+          "incidentManagementService": "ServiceNow",
+          "mappings": {
+            "CustomProperty": "Allowed"
+          },
+          "name": "MyIncidentReceiver"
+        }
+      ]
+    },
```

Comment:
I don'think this maps to the `max` test. If possible, add all these properties to that test, re-run the `Set-AVMModule` script to have it update the readme and then re-run the workflow.